SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -203,6 +204,8 @@ void USpatialSender::CreateServerWorkerEntity(int AttemptCounter)
 		{
 			Sender->NetDriver->WorkerEntityId = Op.entity_id;
 			Sender->NetDriver->GlobalStateManager->TrySendWorkerReadyToBeginPlay();
+			Sender->UpdateWorkerEntityInterestAndPosition();
```

Comment:
We should delete this from here, we have no guarantee that the Translator is ready when this command returns. This is why we wait till `USpatialNetDriver::HandleStartupOpQueueing` and bIsReady is true

Reply:
I would agree, but this is also how we give servers interest over always relevant components. It effectively is called twice, once to get the bare bones interest, then a second time once we know it's ready to create the full fat stuff. I was under the impression that it needs the always relevant interest in order to initialise so left it here- is that not true?

Reply:
Excellent point, well made, stick with it

Reply:
I was wrong

Reply:
we set it at entity construction time